client/app/bundles/course/assessment/submission/actions/annotations.js

```diff
@@ -11,13 +11,20 @@ export function onCreateChange(fileId, line, text) {
   };
 }
 
-export function create(submissionId, answerId, fileId, line, text) {
+export function create(
+  submissionId,
+  answerId,
+  fileId,
+  line,
+  text,
+  delayedComment,
```

Comment:
This seems like a boolean. Can this be renamed to `isDelayedComment`?

Reply:
Noted!